soc/arm/nordic_nrf/nrf53/soc_cpu_idle.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file SoC extensions of cpu_idle.S for the Nordic Semiconductor nRF53 processors family.
+ */
+
+
+#if defined(_ASMLANGUAGE)
+
+#define SOC_ON_EXIT_CPU_IDLE \
+	nop; \
+	nop; \
+	nop; \
+	nop;
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
@carlocaione isn't commit message not enough?

Reply:
I personally think it is easier to read a comment that the git log.